CODEOWNERS

```diff
@@ -1,5 +1,5 @@
 # SPDX-License-Identifier: Apache-2.0
 
 # Fabric Test Maintainers
-*	@lindluni @denyeart @dongmingh @mastersingh24 @scottz64 @suryalnvs
-/tools/chaincode-integration @awjh-ibm @mbwhite
+*	@fabric-release-maintainers
```

Comment:
You didn't want to add fabric-test-maintainers?

Reply:
Whoops, pulled the wrong name